### PR TITLE
Corrections for June 2021

### PR DESCRIPTION
**Pull request by xinru1414**

### CODE REVIEW

**Comment by mjpost** [comment]
I made this a draft since we won't merge till July. Will take a look soon.

**Comment by bekou** [comment]
Hi @xinru1414 , we have closed #1362. The issue has been fixed by google scholar, thus there is no need to update the manuscript.

File: data/xml/2021.naacl.xml [hunk 1]
```diff
@@ -5895,8 +5897,10 @@
       <author><first>James</first><last>Pustejovsky</last></author>
       <pages>76–87</pages>
       <abstract>We propose semantic visualization as a linguistic visual analytic method. It can enable exploration and discovery over large datasets of complex networks by exploiting the semantics of the relations in them. This involves extracting information, applying parameter reduction operations, building hierarchical data representation and designing visualization. We also present the accompanying COVID-SemViz a searchable and interactive visualization system for knowledge exploration of COVID-19 data to demonstrate the application of our proposed method. In the user studies, users found that semantic visualization-powered COVID-SemViz is helpful in terms of finding relevant information and discovering unknown associations.</abstract>
-      <url hash="964dc194">2021.naacl-srw.11</url>
+      <url hash="2e0108f5">2021.naacl-srw.11</url>
       <doi>10.18653/v1/2021.naacl-srw.11</doi>
+      <revision id="1" href="2021.naacl-srw.11v1" hash="964dc194"/>
+      <revision id="2" href="2021.naacl-srw.11v2" hash="2e0108f5" date="2021-06-28">Added a sponsor to Ack section</revision>
```
Comment (mjpost) [inline data/xml/2021.naacl.xml:5903]: I would probably expand this, e.g., "Added a sponsor to the Acknowledgments section."

**Review by mjpost — APPROVED** [review]

**Comment by mjpost** [comment]
Do you want to pull in #1386 and then merge?

**Review by mjpost — APPROVED** [review]
This looks good, apart from a small comment about the wording in one place. It would also be nice to pull in one or two recent ones, then merge.

File: data/xml/2021.naacl.xml (hunk 1 above)
Comment (xinru1414) [reply data/xml/2021.naacl.xml:5903]: done

**Pull request merged**